### PR TITLE
Add missing package names and versions

### LINKED ISSUE #375: Fix Nix Mix
**Describe the bug**
The Nix build is currently reporting warnings like this:

```
trace: warning: crane cannot find name attribute in /nix/store/i70apg8wmqwaw1d20w56yr3hyjvr4f21-source/Cargo.toml, consider setting `pname = "...";` explicitly
trace: warning: crane cannot find version attribute in /nix/store/i70apg8wmqwaw1d20w56yr3hyjvr4f21-source/Cargo.toml, consider setting `version = "...";` explicitly
```

**To Reproduce**

```
nix build
```

**Expected behavior**

No warnings or errors.

**Additional context**

https://github.com/ipetkov/crane/pull/237#issuecomment-1473931031

### PR DESCRIPTION
**Pull request by nbacquey**
Closes #375

### CODE REVIEW

**Review by Xophmeister — APPROVED** [review]
This looks fine to me, but we should wait on a second opinion -- from someone who knows Nix better than me! -- before being merged.

(I will merge in your absence, once approved :ok_hand:)

**Review by torhovland — APPROVED** [review]
Looks good!

**Pull request merged**
